src/core/src/op/multiclass_nms.cpp

```diff
@@ -2,68 +2,118 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/multiclass_nms.hpp"
-
-#include <cstring>
-#include <ngraph/validation_util.hpp>
+#include "openvino/op/multiclass_nms.hpp"
+#include "multiclass_nms_shape_inference.hpp"
 
 #include "itt.hpp"
-#include "ngraph/attribute_visitor.hpp"
-#include "ngraph/op/constant.hpp"
-#include "ngraph/op/util/op_types.hpp"
-#include "ngraph/runtime/reference/multiclass_nms.hpp"
-#include "ngraph/type/bfloat16.hpp"
-#include "ngraph/type/float16.hpp"
-#include "ngraph/util.hpp"
 
-using namespace ngraph;
+using namespace ov;
```

Comment:
This may affect other `cpp` files in same static library because they may use `ngraph` namespace